src/community/blocks.tsx

```diff
@@ -250,7 +250,9 @@ export const ProgressBlock: React.FC<ProgressBlockProps> = ({
                     </Box>
                     {stackingGroup}
                 </Inline>
-                {progressPercent && <ProgressBar progressPercent={progressPercent} reverse={reverse} />}
+                {progressPercent !== undefined && (
```

Comment:
Consider:
`Number.isFinite(progressPercent) && (...)`

Reply:
Discussed offline. We are leaving it as it is for simplicity, because TS type refinement is not working well with `Number.isFinite` and we believe it's not worth it to add extra checks only for this scenario.